ludwig/modules/metric_modules.py

```diff
@@ -123,11 +123,27 @@ def get_inputs(cls):
 
 
 @register_metric(ROC_AUC, [BINARY])
-class ROCAUCMetric(AUROC, LudwigMetric):
-    def __init__(self, **kwargs):
-        super().__init__(dist_sync_fn=gather_all_tensors)
+class ROCAUCMetric(LudwigMetric):
+    """Fast implementation of metric for area under ROC curve."""
 
-    """ Metric for area under ROC curve. """
+    def __init__(
+        self,
+        num_thresholds: int = 201,
+        epsilon: float = 1e-7,
+        compute_on_step: Optional[bool] = None,
+        dist_sync_on_step: bool = False,
+        process_group: Optional[Any] = None,
+        dist_sync_fn: Optional[Callable] = None,
```

Comment:
Should be `gather_all_tensors` (see `MeanMetric` just below).

Reply:
Done!